src/app/locales/ru/routes.yml

```diff
@@ -0,0 +1,181 @@
+error:
+    header: Ошибка
+    text: при попытке отобразить эту страницу произошла ошибка.
+about:
+    about:
+        header: о нас
+        homepage: домашняя страница
+        author: автор
+        version: Версия
+        nwjs: NW.js
+        arch: Архитектура
+        built: построен
+        help: Помошь
+        wiki: Wiki
+        bugs: ошибки
+        issue-tracker: Система отслеживания ошибок
+        support: Служба поддержки
+        gitter: Gitter
+        changelog: Изменения
+        contributors: Авторы
+    donation:
+        header: Поддержка и пожертвование
+        text: Если вы считаете, что это приложение полезно, попробуйте поддержать его создателя, сделав пожертвование.<br>Большое спасибо!
+    translations:
+        header: Переводы
+    dependencies:
+        header: Сторонние приложения
+featured:
+    header: Избранные каналы
+    empty: Список рекомендуемых каналов не может быть загружен!
+    summary: {{viewers}} зрителей смотрят трансляцию {{channels}} прямо сейчас !
```

Comment:
did I change correctly in package.json?

Reply:
> did I change correctly in package.json?

You didn't push any changes to the file. Just leave it as it is now. It doesn't really matter, as all syntax errors should be fixed if you push the change I've just suggested. It would only have helped to figure out new issues, but that's not important now.

Reply:
Btw, if you locally run `yarn run grunt webpack:i18n`, `yarn run grunt test` or `yarn run grunt build`, you'll see if there are errors left (before pushing).

Reply:
I did not start building the application